### ISSUE TITLE
MLP

### ISSUE THREAD

**Issue by Mohammed-Ryiad-Eiadeh** [oracle/tribuo#351]
Dear,

I want to use MLP with only one hidden layer of 50 units and the output layer (1 unit since my problem is binary classification and sigmoid fun). And I want to use universal MLP rule (strengthen and weaken the neuron) without any kind of optimizer like adagrad and I need to modify the number of learning rate for different rounds. Is this possible in Tribuo?!

Warm Regards.

**Comment by Craigacp** [comment]
We don't have the perceptron learning rule implemented, and haven't released the neural net package prototype we have because the math library isn't multithreaded yet (so it's very slow).

We are working on more control over the termination condition in SGD, and modifying the learning rate should be easy by adding a new optimizer, but the rest would have to be implemented.
